app/src/main/AndroidManifest.xml

```diff
@@ -70,6 +77,13 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/shared_file_paths" />
         </provider>
+
+        <meta-data
+            android:name="com.google.android.gms.version"
+            android:value="@integer/google_play_services_version" />
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="@string/google_api_key"/>
```

Comment:
What happens if this is unset? Does it need to be added to dummy-google-services.json? or some other dummy file?

Reply:
It's the client.api_key there. Some magic somewhere interprets this into that AFAIK